README.md

```diff
@@ -0,0 +1,35 @@
+# AI-Remember 📔
+
+## About AI-Remember 🌟
+
+AI-Remember simplifies the process of recording cherished memories through diaries. It offers a user-friendly approach, sparing users the effort of writing lengthy diaries themselves, which can often be time-consuming. AI-Remember steps in as a supportive companion, gently guiding conversations with users to ensure that no detail or sentiment is left unexplored. At the end of each day, it compiles all the stories shared by the user into a beautifully summarized diary, transforming precious moments into lasting memories.
+
+## Group Members 👥
+
+| Matriculation Number | Name                | Contributions                             |
+| -------------------- | ------------------- | ----------------------------------------- |
+| A0237732A | Xu Yi        | [Description of contributions] |
+| [Matriculation Number 2] | [Member 2]        | [Description of contributions by Member 2] |
+| [Matriculation Number 3] | [Member 3]        | [Description of contributions by Member 3] |
+
+## Application URL 🌐
+
+https://florentianayuwono.github.io/ai-remember/
+
+## Set-up Instructions for Local Testing ⚙️
+
+If you want to test AI-Remember locally, follow these steps:
+
+1. Clone the repository to your local machine:
+   ```bash
+   git clone https://github.com/florentianayuwono/ai-remember.git
+2. Navigate to the project directory:
+   ```bash
+   cd ai-remember
+3. Install dependencies:
+    ```bash
+    npm install
+4. Start the local preview:
+    ```bash
+    npm run preview
```

Comment:
Can also add these alternatives:
`npm run dev` for running dev server, `npm run build` to build the app, `npm run preview` for running an already built app as a production preview. Usually I just do `npm run dev` to see hot live changes since it's not ready for build yet most of the times haha.